examples/machine_translation/char_dataset.py

```diff
@@ -0,0 +1,121 @@
+import itertools
+import re
+
+import torch
+from torch.utils.data import DataLoader
+
+from torchtext.data.utils import get_tokenizer
+from torchtext.experimental.datasets.translation import (DATASETS,
```

Comment:
You should be able to load the raw datasets from `torchtext.experimental.datasets.raw` directly.

Reply:
So just to be clear, it's not recommended/intended to use the raw datasets from `torchtext.experimental.datasets`?